index.js

```diff
@@ -0,0 +1,103 @@
+const element = document.getElementById('nav-button');
+const mobileMenu = document.getElementById('mobileMenu');
+const close = document.getElementById('close');
+element.addEventListener('click', () => {
+  const x = document.getElementById('myLinks');
+  if (x.style.display === 'block') {
+    element.innerHTML = '<I class="fa fa-bars"></I>';
+    x.style.display = 'none';
+    window.location.reload(false);
```

Comment:
- [Optional] Please try to use thesame space between the cancel icon (X) and the portfolio link as it is in the figma design. 

##### Your design
![Screenshot from 2022-07-23 22-49-32](https://user-images.githubusercontent.com/58771507/180623984-92441a27-e838-499a-b444-8f58d0b49b18.png)

##### Figma design

![Screenshot from 2022-07-23 22-53-05](https://user-images.githubusercontent.com/58771507/180624056-7782019f-1eba-469f-a4c1-11a242f12dc0.png)